client/state/sites/reducer.js

```diff
@@ -268,6 +268,17 @@ export const deleting = keyedReducer(
 	)
 );
 
+/**
+ * Tracks whether all sites have been fetched.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action object
+ * @return {Object}        Updated state
+ */
+export const hasAllSitesList = createReducer( false, {
+	[ SITES_RECEIVE ]: () => true,
+} );
```

Comment:
Is the `hasAllSitesList` properly serialized and deserialized from the local storage? It would be unfortunate if there was a full site list in local storage and then it wasn't used because `hasAllSitesList` is not stored and was initialized to `false`.

I think you need to pass a schema to `createReducer` to turn on persistence. But I'm not 100% sure.

Reply:
Good catch, thanks. Fixed in b8f84ae.